app/src/main/java/com/activities/CameraActivity.java

```diff
@@ -36,6 +36,8 @@
     private CameraBridgeViewBase mOpenCvCameraView;
     private FrameProcessTask frameProcessTask;
     private Button takePhotoButton;
+    private boolean saveNextFrame = false;
+    private String TAG = "CameraActivity";
```

Comment:
I have changed "TAG" to "Tag" in other places is the code to make naming unified. Also I think it should be final String.